modules/WebServer.py

```diff
@@ -4,29 +4,47 @@
 server = None
 web_server_ip = "0.0.0.0"
 web_server_port = "8000"
+web_server_template = "www"
 
 
-def initialize_web_server(config):
-    import threading
-    global web_server_ip, web_server_port
-
-    if config.has_option('BOT', 'customWebServerAddress'):
-        custom_web_server_address = (config.get('BOT', 'customWebServerAddress').split(':'))
-        if len(custom_web_server_address) == 1:
-            custom_web_server_address.append("8000")
-            print "WARNING: Please specify a port for the webserver in the form IP:PORT, default port 8000 used."
+def get_web_server_setting(config, option, default):
+    '''
+    Retrieve specific configuration from config file
+    '''
+    if config.has_option('BOT', option):
+        value = config.get('BOT', option)
     else:
-        custom_web_server_address = ['0.0.0.0', '8000']
+        value = default
+    return value
+
+def initialize_web_server(config):
+    '''
+    Setup the web server, retrieving the configuration parameters
+    and starting the web server thread
+    '''
+    global web_server_ip, web_server_port, web_server_template
+
+    # Check for custom web server address
+    web_server_ip = get_web_server_setting(config, 'customWebServerAddress', '0.0.0.0')
```

Comment:
Can we find an elegant way not to break current config, or at least issue an error if web_server_ip contains the port information as it is in the current config?